storybook/src/components/TextInput/TextInput.docs.mdx

```diff
@@ -26,7 +26,7 @@ On some devices, it may show an email-specific keyboard.
 Consider setting the following attributes:
 
 1. Set `autocomplete="email"` to help browsers autofill the user’s email address.
-2. Set `autocapitalize="none"`, `autocorrect="off"` and `spellcheck="false"` to stop browsers automatically changing user input.
+2. Set `autocapitalize="off"`, `autocorrect="off"` and `spellcheck="false"` to stop browsers automatically changing user input.
```

Comment:
Apple recommends using `none`, they deprecated `off`

Reply:
Are you an encyclopaedia? (Serious question.) (Fixed.)

Reply:
Ha! No, just crazy Google skills 😁